src/_includes/site-nav.html

```diff
@@ -1,12 +1,22 @@
-<nav class="site-nav">
-  <ul class="u-list-reset">
+<nav class="site-nav js-toggle-menu">
+  <button type="button" class="u-button-reset js-toggle-trigger" aria-controls="site-menu">
+    <span aria-hidden="true" class="menu-icon">
+      <div class="menu-icon__line"></div>
+      <div class="menu-icon__line"></div>
+      <div class="menu-icon__line"></div>
+    </span>
+    Menu
+  </button>
+  <ul id="site-menu" class="u-list-reset">
     {% assign nav = collections.nav | sortByPageOrder %}
     {% for item in nav %}
-    <li{% if page.url == item.url %} aria-current="page" {% endif %}>
+    <li {% if page.url == item.url %} aria-current="page" {% endif %}>
       <a href="{{ item.url }}">
         {{ item.data.pageTitle }}
       </a>
-      </li>
-      {% endfor %}
+    </li>
+    {% endfor %}
   </ul>
 </nav>
+<!--       <script src="menu-bundled.js"></script> -->
+<script src="menu.js"></script>
```

Comment:
This `script` is missing the folder, so it's never loading on your page, which results in the hamburger menu not working:

```suggestion
<script src="/js/menu.js"></script>
```

Also, to get any of the JS files in that directory to work, you also need to add the following:
`eleventyConfig.addPassthroughCopy("./src/js/");` to your `eleventy.js` file (it can go on a new line after the `setTemplateFormats` function).